docs/specification/1.0.md

```diff
@@ -13,6 +13,16 @@ The CDS Hooks specification is available as an Open API Specification using Swag
 
 ## Discovery
 
+Developers of CDS Services MUST provide a stable endpoint for allowing EHRs to discover available CDS Services, including information such as the purpose of the CDS Service, when it should be invoked, and any data that is requested to be prefetched.
+
+A CDS service MAY expose its services at 
```

Comment:
Tagging onto what Bryn pointed out, perhaps we reword this simply to "A CDS Service provider MUST expose its Discovery endpoint at"